whelk-core/src/main/groovy/whelk/converter/marc/RomanizationStep.groovy

```diff
@@ -62,9 +62,13 @@ class RomanizationStep extends MarcFramePostProcStepBase {
     List FIELD_REFS = [FIELDREF, BIB250_REF]
 
     void modify(Map record, Map thing) {
+        def bib880 = thing[HAS_BIB880]
         try {
             _modify(record, thing)
         } catch (Exception e) {
+            if (bib880) {
+                thing[HAS_BIB880] = bib880
+            }
             log.error("Failed to convert 880: $e", e)
```

Comment:
```suggestion
            log.error("Failed to convert 880 for record ${record[ID]}: $e", e)
```

Reply:
Wasn't sure if we have a record id at this stage...do we?